peripherals/usart/_v2_AB_common.yaml

```diff
@@ -0,0 +1,21 @@
+# Common fields between v2 A and B peripherals
+
+CR1:
+  UESM:
+    Disabled: [0, "USART not able to wake up the MCU from Stop mode"]
+    Enabled: [1, "USART able to wake up the MCU from Stop mode"]
+CR3:
+  WUFIE:
+    Inhibited: [0, "Interrupt is inhibited"]
+    Interrupted: [1, "An USART interrupt is generated whenever WUF=1 in the ISR register"]
```

Comment:
I think these two should be `Enabled` and `Disabled` to match other `*IE` registers like `TXEIE`?